### ISSUE TITLE
i would like to create a new telegraf config from the UI and edit it

### ISSUE THREAD

**Issue by russorat** [influxdata/ui#786]
We have a way to create telegraf configurations in the UI today, but clicking the button opens a modal with just a few options:
![image](https://user-images.githubusercontent.com/4805997/110149457-e96f1300-7d92-11eb-8b2e-2c5eff073739.png)

![image](https://user-images.githubusercontent.com/4805997/110149477-f12eb780-7d92-11eb-8f0d-1210dc449600.png)

Now that we can manually edit telegraf configs in the UI, we should give the user the option to create a new telegraf config without going through the modal.

### Acceptance Crit
as a user, i would like the ability to create a new telegraf configuration that i can edit manually
as a user, i would like to have the new telegraf config configured to write to my cloud instance already
as a user, i would like a way to customize the telegraf config before i save it

### Notes
 - Telegraf has an option to generate a new config with the `telegraf config` command. That should match https://github.com/influxdata/telegraf/blob/master/etc/telegraf.conf
 - By default, it has the `outputs.influxdb` configured, but in this case, we want the `outputs.influxdb_v2` configured and ready to go
 - Related to https://github.com/influxdata/ui/issues/784
 - I'm ok completely replacing the existing "create telegraf" experience in the app.

**Comment by sjwang90** [comment]
> I'm ok completely replacing the existing "create telegraf" experience in the app.

I agree. Possibly not even having to remove the current capabilities of the 5 "Automatically configured" plugins but move that experience within the list of plugins total. Ex: I can search for the `system` plugin in listed of data sources, i can pointy click my way to create a Telegraf + config for it.
